apps/andi/assets/css/edit.scss

```diff
@@ -6,10 +9,16 @@
   padding: 2rem;
 }
 
+.form-component {
+    margin-right: 3rem;
+}
+
 .form-section {
-  border: $table-border;
-  padding: 2rem;
-  margin-bottom: 4rem;
+  padding-left: $component-number-width - 1rem;
+  margin-top: -$component-header-height;
+  padding-top: $component-header-height;
+  margin-left: 1rem;
+  border-left: 2px solid #C9C7C7;
```

Comment:
#C9C7C7 seems to be used a lot. Maybe we should make it a sass variable.

Reply:
true